src/rdkafka_conf.c

```diff
@@ -952,7 +952,7 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
      "is used. If set it to \"oidc\", OAuth/OIDC login method will "
      "be used. "
      "sasl.oauthbearer.client.id, sasl.oauthbearer.client.secret, "
-     "sasl.oauthbearer.scope, sasl.oauthbearer.extensions, "
+     "sasl.oauthbearer.scope, "
```

Comment:
Hi @edenhill , do we need to separate this description for standard AK and CC?

Reply:
@jliunyu No I don't think so. The ccloud config requirements should not be expressed here but in ccloud docs&examples.